stacks-signer/src/runloop.rs

```diff
@@ -31,6 +32,11 @@ pub enum RunLoopCommand {
         /// Taproot merkle root
         merkle_root: Option<MerkleRoot>,
     },
+    /// Send this Command to the runlop to start an RTT broadcast.
+    Ping {
+        /// Attach a payload of random bytes to the Ping/Pong messages.
+        payload_size: u16,
```

Comment:
The payload can be considerably bigger than 2^16 bytes.  The maximum payload size is 16 MB.